pkg/python/env.go

```diff
@@ -32,5 +32,10 @@ func CreateVirtualEnvironment(
 	if err := executeFn("bash", "-c", fmt.Sprintf("source %s", filepath.Join(virtualEnv, "bin", "activate"))); err != nil {
 		return exitHandler, fmt.Errorf("failed to activate virtual environment %s: %w", virtualEnv, err)
 	}
+	pipPath := filepath.Join(virtualEnv, "bin", "pip")
+	if err := executeFn(pipPath, "install", "--upgrade", "pip", "build", "wheel", "setuptools"); err != nil {
+		return exitHandler, fmt.Errorf("failed to activate virtual environment %s: %w", virtualEnv, err)
+	}
```

Comment:
There are functions available to install `wheel` and `build` we could reuse. However these are single python calls.

https://github.com/SAP/jenkins-library/blob/dc940097d4438b7cef60031ec87375a167b5db4d/pkg/python/build.go#L43-L48

Also it is later installed in toml build again. Maybe we should unify instead.